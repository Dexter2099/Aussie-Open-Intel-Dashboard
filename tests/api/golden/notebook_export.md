# My Notebook

Created by: anonymous
Created at: 2024-01-01T12:00:00+00:00

- [Event] Event Alpha (2024-01-02T08:30:00+00:00) https://example.com/src â€” first
- [Entity] Org: ACME

