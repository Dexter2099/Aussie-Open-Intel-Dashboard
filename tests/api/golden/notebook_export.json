{
  "id": "11111111-1111-4111-8111-111111111111",
  "title": "My Notebook",
  "created_by": "anonymous",
  "created_at": "2024-01-01T12:00:00+00:00",
  "items": [
    {
      "kind": "event",
      "id": "22222222-2222-4222-8222-222222222222",
      "title": "Event Alpha",
      "time": "2024-01-02T08:30:00+00:00",
      "source_url": "https://example.com/src",
      "note": "first",
      "created_at": "2024-01-01T12:00:00+00:00"
    },
    {
      "kind": "entity",
      "id": "33333333-3333-4333-8333-333333333333",
      "type": "Org",
      "name": "ACME",
      "note": null,
      "created_at": "2024-01-01T12:00:00+00:00"
    }
  ]
}

